### ISSUE TITLE
isOffFrame goes true when fully zoomed out

### ISSUE THREAD

**Issue by frodberserk** [TakuSemba/CropMe#20]
The flag isOffFrame() returns true sometimes when the image is fully zoomed out. The box is still containing the part of image and I don't see the frame going off the image. Why is this happening?

My xml code:
```
<com.takusemba.cropme.CropLayout
        android:id="@+id/pvImage"
        android:layout_width="0dp"
        android:layout_height="0dp"
        android:layout_margin="@dimen/screen_horizontal_margin"
        android:background="@color/colorBlack"
        android:src="@drawable/bg_splash"
        app:cropme_background_alpha="80%"
        app:cropme_frame_height_percent="75%"
        app:cropme_frame_width_percent="75%"
        app:cropme_max_scale="2.0"
        app:cropme_overlay_shape="rectangle"
        app:cropme_with_border="true"
        app:layout_constraintBottom_toTopOf="@id/btnChangePhoto"
        app:layout_constraintDimensionRatio="1:1"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toBottomOf="@id/tvHeading" />
```

**Comment by ViktorPetrovski** [comment]
I'm having the same issue on android 11, any fix?

**Comment by ViktorPetrovski** [comment]
@frodberserk Try setting 
```
app:cropme_frame_height_percent="75%"
        app:cropme_frame_width_percent="75%"
``` to 80% that did the trick for me. Still can't understand why

**Comment by silwek** [comment]
I have the same issue, it's because of a float to int on frameRect. 
In my case, `frameRect.left=161.999` when `targetRect.left=162` but `frameRect.left.toInt()=161`. So off frame...

I found no easy way to fix it.
I'll try a fork with something like this :
```
fun isOffFrame(): Boolean {
    val frameRect = frameCache ?: return false
    val targetRect = Rect()
    cropImageView.getHitRect(targetRect)
    return !targetRect.contains(
            ceil(frameRect.left.toDouble()).toInt(),
            ceil(frameRect.top.toDouble()).toInt(),
            floor(frameRect.right.toDouble()).toInt(),
            floor(frameRect.bottom.toDouble()).toInt()
        )
}
```

**Comment by TakuSemba** [comment]
I included this fix in v2.0.8. https://github.com/TakuSemba/CropMe/pull/22
Let me know if it still happens or any issue. :)
